### PR TITLE
CONTRIBUTING.md: more description of changelogs

### PR DESCRIPTION
**Pull request by ulysses4ever**
In particular, mention that Markdown is allowed.

---
Please include the following checklist in your PR:

* [N/A] Patches conform to the [coding conventions](https://github.com/haskell/cabal/blob/master/CONTRIBUTING.md#other-conventions).
* [N/A] Any changes that could be relevant to users [have been recorded in the changelog](https://github.com/haskell/cabal/blob/master/CONTRIBUTING.md#changelog).
* [x] The documentation has been updated, if necessary.
* [N/A] Include [manual QA notes](https://github.com/haskell/cabal/blob/master/CONTRIBUTING.md#qa-notes) if your PR relates to cabal-install.

Bonus points for added automated tests!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -242,13 +242,16 @@ and concatenate the commit messages of the commits before merging.
 Changelog
 ---------
 
-When opening a pull request, you should write a changelog entry
-(or more in case of multiple independent changes).
-This is done by adding files in the `changelog.d` directory.
+When opening a pull request with a user-visible change, you should write a changelog entry
+(or more in case of multiple independent changes) -- the desciption will end up in
```
Comment (ffaf1) [inline CONTRIBUTING.md]: — ← em dash instead of `--` maybe (like you did below)?

**Review by geekosaur — COMMENTED** [review]
Looks reasonable to me, not that I count as a reviewer.

**Review by ffaf1 — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (ulysses4ever) [reply CONTRIBUTING.md]: thanks, already fixed!

**Review by Mikolaj — APPROVED** [review]
LGTM

**Pull request merged**